### PR TITLE
translation for '.github' and sermant-agentcore-config

### PR DESCRIPTION
**Pull request by lilai23**
**What type of PR is this?**

Doc

**What this PR does / why we need it?**

This initiative aims to enhance readability, accessibility, and collaboration for both current and potential contributors, ensuring a unified language throughout the codebase.

**Which issue(s) this PR fixes？**

Fixes #1466

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-agentcore/sermant-agentcore-config/config/logback.xml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-    <!-- %L为行号-->
-    <!-- %thread线程名称-->
-    <!-- %m或者%msg为信息-->
-    <!-- %n换行-->
+    <!-- Configure the log output format -->
+    <!-- %d{yyyy-MM-dd HH:mm:ss.SSS} for date-->
+    <!-- %C for class name-->
+    <!-- %F for file name-->
+    <!-- %M for method name-->
+    <!-- %L for line-->
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-config/config/logback.xml]: for line number

Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-config/config/logback.xml]: done

File: sermant-agentcore/sermant-agentcore-config/config/config.properties [hunk 2]
```diff
@@ -1,98 +1,98 @@
...
+# Service name for host instance
 service.meta.service=default
-# 指定应用名称, 用于服务注册等服务治理场景
+# Application name for host instance, used in service governance scenarios such as service registration and label routing
 service.meta.application=default
-# 指定服务版本, 用于服务注册、标签路由等服务治理场景
+# Version for host instance, used in service governance scenarios such as service registration and label routing
 service.meta.version=1.0.0
-# 指定服务命名空间, 用于服务注册等服务治理场景
+# Project for host instance, used in service governance scenarios such as service registration and label routing
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-config/config/config.properties]: Namespaces for host instance

File: sermant-agentcore/sermant-agentcore-config/config/config.properties [hunk 3]
```diff
@@ -1,98 +1,98 @@
...
 service.meta.project=default
-# 指定服务所在环境, 用于服务注册等服务治理场景
+# Environment for host instance, used in service governance scenarios such as service registration and label routing
 service.meta.environment=
-# 指定服务所在az（可用区）, 用于服务注册、标签路由等服务治理场景
+# Zone for host instance, used in service governance scenarios such as service registration and label routing
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-config/config/config.properties]: Availability Zone for host instance

Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-config/config/config.properties]: Available Zone

File: sermant-agentcore/sermant-agentcore-config/config/config.properties (hunk 2 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-config/config/config.properties]: done

**Review by hanbingleixue — DISMISSED** [review]

**Review by daizhenyu — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Pull request merged**